certifier/templates/certifier/certify.html

```diff
@@ -1,291 +1,392 @@
-                            <option selected>Choose...</option>
-                            <option value="timing_scan">Timing scan</option>
-                            <option value="hv_problem">HV Problem</option>
-                            <option value="misconfiguration">Misconfiguration</option>
-                            <option value="other">Other</option>
-                        </select>
-
+                    <div class="row no-gutters">
+                        <div class="col-sm-3 col-form-label"style="font-size: 19px;">
+                            {{ form.pixel_problems.label}}:
+                        </div>
+                        <div class="col-sm-5">
+                            {% bootstrap_field form.pixel_problems field_class="form-control problems" label="" %}
+                        </div>
+                    </div>
+                </div>
+                <div class="col mt-3">
+                    <div class="row no-gutters">
+                        <div class="col-sm-3 col-form-label"style="font-size: 19px;">
+                            {{ form.strip_problems.label}}:
+                        </div>
+                        <div class="col-sm-5">
+                            {% bootstrap_field form.strip_problems field_class="form-control problems" label="" %}
+                        </div>
+                    </div>
+                </div>
+                <div class="col mt-3">
+                    <div class="row no-gutters">
+                        <div class="col-sm-3 col-form-label"style="font-size: 19px;">
+                            {{ form.tracking_problems.label}}:
+                        </div>
+                        <div class="col-sm-5">
+                            {% bootstrap_field form.tracking_problems field_class="form-control problems" label="" %}
+                        </div>
+                    </div>
+                </div>
+<!--================================ COMMENT ============================-->
+                <div class="col my-2">
+                    <div class="row no-gutters">
+                        <div class="col-sm-12 col-form-label" style="font-size: 19px;">
+                            {{ form.comment.label}}:
+                        </div>
+                        <div class="col-sm-8">
+                        {% render_field form.comment class="form-control" rows="4" %}
+                        </div>
                     </div>
                 </div>
-                <div class="col"></div>
             </div>
+        </div>
+    </form>
+</div>
 
-            <div class="row mt-5 justify-content-center">
-                <input class="btn btn-primary btn-lg" type="submit" value="Submit">
-            </div>
+<!--================================ SCRIPTS ==============================-->
+<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
```

Comment:
This would make two versions of jquery running at the same time. Did you check if they cause any conflicts?